packages/airnode-validator/src/api/api.ts

```diff
@@ -45,26 +45,15 @@ const NO_MATCH_REGEXP = /($^)/;
 const ES_MATCH_REGEXP = /\$\{([^\\}]*(?:\\.[^\\}]*)*)\}/g;
 
 function interpolateSecrets(config: unknown, secrets: Secrets): ValidationResult<unknown> {
-  // TODO: Replace with go utils
```

Comment:
This was easy to implement. It also solved https://api3dao.atlassian.net/browse/AN-709